skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +474,21 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        rpix, phipix = get_r_phi(shape, origin)
```

Comment:
maybe I misunderstand, why can't you just flip to (colc, rowc), (shape[0], shape[1]), doesn't that achieve the same thing? 
I might be missing something. The only case I'd see it mattering is when the fastest varying index by convention is not the rightermost index (assuming contiguous 2D array in memory)? I could see that not always being true in numpy given you can access the same memory using either `a` or `a.T`. I'm interested to learn why this matters here.
In either case, if the striding/fastest index matters, could you also do this maybe?:
```
rpix = radial_grid((rowc,colc), (shape[0],shape[1])).T
```